src/kitovu/sync/syncing.py

```diff
@@ -91,7 +91,12 @@ def _sync_subject(subject: typing.Dict[str, str],
     remote_dir = pathlib.PurePath(subject['remote-dir'])  # /Informatik/Fachbereich/EPJ/
     local_dir = pathlib.Path(subject['local-dir'])  # /home/leonie/HSR/EPJ/
 
+    ignore = subject['ignore']
+
     for remote_full_path in plugin.list_path(remote_dir):
+        if remote_full_path.parts[-1] in ignore:
```

Comment:
You should be able to use `.name` instead of `.parts[-1]` ([docs](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.name))